electron/static/splash.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
Did this go through the product team?

Reply:
No, just a placeholder. Removed the funky animation.